packages/tests/tests/peer_exchange.node.spec.ts

```diff
@@ -70,16 +75,29 @@ describe("Peer Exchange", () => {
       expect(peerInfos.length).to.be.greaterThan(0);
       expect(peerInfos.length).to.be.lessThanOrEqual(numPeersToRequest);
       expect(peerInfos[0].ENR).to.not.be.null;
+      expect(peerInfos[0].ENR?.peerInfo?.multiaddrs).to.not.be.null;
 
-      const doesPeerIdExistInResponse =
-        peerInfos.find(
-          ({ ENR }) => ENR?.peerInfo?.id.toString() === nwaku1PeerId.toString()
-        ) !== undefined;
+      const foundNodeMultiaddrs: Multiaddr[] = [];
+      const foundNodePeerId: PeerId | undefined = undefined;
```

Comment:
these values never get set, so that it will always `throw` at `:87` and if not it will `dial` empty `maddrs` array

Reply:
merging this PR now, let me know if you want to suggest any other change & I can follow up